tests/test_ffmpeg.py

```diff
@@ -82,12 +82,13 @@ def test_extract_frames() -> None:
 	]
 
 	for target_path, trim_frame_start, trim_frame_end, frame_total in test_set:
-		create_temp_directory(target_path)
+		output_path = get_test_output_file(f'test-extract-frames-{trim_frame_start}-{trim_frame_end}.mp4')
```

Comment:
we are not using f' in our project

typically these files are named after the method name - see other tests